### DISCUSSION TITLE
OpenSSL Integration with Ulfius

### DISCUSSION THREAD

**Discussion by gayathriaccount** [babelouest/ulfius#255] [Q&A] [answered]
Hi,
    We would like to use OpenSSL rather than GNU TLS for supporting few TLS features along with Ulfius. Is this integration possible? If so, could you please guide us on how we could achieve it.

Thanks and regards
Gayathri

**Comment by babelouest** [comment, marked as answer]
Hello,

Ulfius doesn't implement the TLS layer, it's libmicrohttpd underneath who does. So the OpenSSL integration should happen there instead of Ulfius.

I remember a [question about this subject](https://lists.gnu.org/archive/html/libmicrohttpd/2015-11/msg00014.html) a few years ago, but I don't know if something happened after that.
